GitCommitsForNonGitUsers.md

```diff
@@ -0,0 +1,42 @@
+#Git Commits For Non-Git Users
+
+This is a guide to aid others not familiar with git to contribute to our TIL repo!
+
+####FIRST TIME SETUP
+* copy clone to clipboard  
+![](http://images.thoughtbot.com/TIL/copyClone_s.jpg)
+* in Terminal (as are all `commands` that follow) `git clone <PASTE CLIPBOARD>`
+
+####TO SUBMIT
+* `git pull origin master`  
+* `git checkout -b <BRANCHNAME>`  
+* add files/folders locally
+* `git add .` to add everything in cur dir  
+* `git commit`  
+* write title & message  
```

Comment:
Is $VISUAL set to vim by default on OS X? Can we recommend something else?

    env GIT_EDITOR=TextEdit git commit